docs/howto/operate/ci-cd/aws.md

```diff
@@ -1,147 +0,0 @@
-(ci-cd:aws)=
```

Comment:
I noticed you removed the whole section, maybe there are some pieces we want to keep that could be explanatory for future readers? @choldgraf, thoughts about this one?

Reply:
hmmm - I think you all are the best to judge this. I think that the deleted page mostly focused on cluster set-up, which is now covered in `new-cluster/aws.md`, but is there anything missing?